LibreTransmitterUI/Views/Setup/Libre2DirectSetup.swift

```diff
@@ -127,17 +107,26 @@ struct Libre2DirectSetup: View {
 
         }) {
             VStack(spacing: 10) {
-                Button("Pair Sensor & connect") {
+                Button {
                     pairSensor()
+                } label: {
+                    if isPairing {
+                        HStack(spacing: 10) {
+                            ProgressView()
+                            Text(LocalizedString("Pairing...", comment: "Button title for pairing sensor when pairing"))
```

Comment:
If you cancel a pairing, this text will still be greyed out with the label "Pairing". We probably need a cancelpublisher (or change the parameters of the ppairingService.publisher to return a result even when no sensor has been paired)

Reply:
When I cancel pairing (using the demo CGM), the UI goes away and doesn't stay greyed out.  What are the steps to reproduce this?

Reply:
Enter settings view, change sensor and select libre 2 and then pair. Always reproducible. https://photos.app.goo.gl/uZwNSqCPbDsmZXqJ8

Reply:
Ok, this should be fixed now. Thanks!